lib/identity_cache/should_use_cache.rb

```diff
@@ -0,0 +1,13 @@
+module IdentityCache
+  module ShouldUseCache
+    extend ActiveSupport::Concern
+
+    included do
+      unless self.respond_to?(:should_use_cache?)
+        def self.should_use_cache?
```

Comment:
What method are you worrying about overriding?  The class that calls `include IdentityCache` just needs to define the override after that include.